app/javascript/src/components/Dashboard/Contacts/utils.jsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import { MenuHorizontal } from "neetoicons";
+import { Dropdown, Avatar, Typography } from "neetoui";
+
+const renderNameRoleAvatar = (name, { role }) => (
+  <div className="flex space-x-4">
+    <Avatar
+      user={{
+        imageUrl: "https://xsgames.co/randomusers/avatar.php?g=pixel",
+      }}
+    />
+    <div>
+      <Typography style="h5">{name}</Typography>
+      <Typography className="neeto-ui-text-gray-600" style="body3">
+        {role}
+      </Typography>
+    </div>
+  </div>
+);
+
+const renderDropdown = () => (
+  <Dropdown buttonStyle="text" icon={MenuHorizontal} strategy="fixed">
+    <li>Edit</li>
+    <li>Delete</li>
```

Comment:
https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663761829787419?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

Reply:
@yedhink  When I am using `MenuItem.Button ` in place of `li` . I am getting an error in the console.
<img width="553" alt="Screenshot 2022-09-23 at 4 16 51 PM" src="https://user-images.githubusercontent.com/43422638/191944805-e023a77c-a7b1-4ea8-94f6-9c081c0e19bb.png">

The changes that I made to `renderDropdown`

`const renderDropdown = () =>{
  const { Menu, MenuItem } = Dropdown;
  return(
    <Dropdown buttonStyle="text" icon={MenuHorizontal} strategy="fixed">
      <Menu>
        <MenuItem.Button style="danger">Delete</MenuItem.Button>
        <MenuItem.Button>Edit</MenuItem.Button>
      </Menu>
    </Dropdown>
)};`

Please help me out with this.

Reply:
Updating neetoUI to V4 solved the issue. I will be raising a new pull request with all the changes according to the review.

Reply:
I was going to link https://github.com/Ishaan-Singhal/neetoUI-challenge/pull/9#discussion_r978600442. But since you already fixed the issue, no need to redo anything.